clayui.com/src/components/IconSearch/IconSearch.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * © 2020 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayForm, {ClayInput} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import React, {useEffect, useState} from 'react';
+
+import icons from './icons.json';
+
+const spritemap = '/images/icons/icons.svg';
+
+export default () => {
+	const [searchQuery, setSearchQuery] = useState('');
+	const [filteredIconNames, setFilteredIconNames] = useState([]);
+
+	const handleIconSearch = event => {
+		setSearchQuery(event.target.value);
+	};
+
+	useEffect(() => {
+		const filteredIcons = icons.filter(icon => {
+			if (icon.aliases.includes(searchQuery)) {
```

Comment:
Instead of using an `if` block here, we can simplify this with an implicit return.

```js
const filteredIcons = icons.filter(
     icon => icon.aliases.includes(searchQuery)
);
```

Reply:
Also, we should additionally filter based off of the icon name itself. For example, if someone wants to search for "add" the `add-cell` icon won't actually show up since we are only searching based off its aliases.

![Screen Shot 2020-02-03 at 10 41 28 AM](https://user-images.githubusercontent.com/6843530/73684797-bb903a00-4679-11ea-9858-fe4f7f911675.png)